shop/src/pages/Product.js

```diff
@@ -37,6 +37,28 @@ function getImageForVariant(productData, variant) {
   }
 }
 
+/**
+ * Finds and returns the variant that has the least price
```

Comment:
nit: the "cheapest" price vs the "least" price

Reply:
Thanks @franckc, Have to work on my vocabulary.